KaijuKiller/Assets/Scripts/PlayerMove.cs

```diff
@@ -76,6 +78,11 @@ void Update()
 
     }
 
```

Comment:
There is potential for this to break things but given the context of your game its probably ok for now